components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/impl/APIUsageStatisticsRdbmsClientImpl.java

```diff
@@ -2336,6 +2347,13 @@ public boolean isTableExist(String tableName, Connection connection) throws SQLE
      */
     @Override
     public void deployArtifacts(String url, String user, String pass) throws Exception {
+
+        if (url.trim().equals("")) {
```

Comment:
yes. it should be flip and since this scenario was not valid it is removed in commit: https://github.com/wso2/carbon-apimgt/commit/006d2e055000123aa5f1e15f2b21a85b8003b621